README_INTERFACE_TESTS.md

```diff
@@ -0,0 +1,135 @@
+# Interface tests
```

Comment:
This page is pretty hard to follow. It serves as a tutorial, reference, explanation, how-to but without clear delineations. 

You may want to make those separations clearer with subections called "tutorial (or getting started)" and "explanation" or simply get rid of the exmplanations and focus on the tutorial.

Reply:
I'll get started on cleaning it up, meanwhile could you check that the remaining issues are resolved?

Reply:
there should be a decent howto now.